docs/sources/tutorials/get-started.md

```diff
@@ -14,21 +14,40 @@ You don't need to know about the {{< param "PRODUCT_NAME" >}} [configuration syn
 
 ## Prerequisites
 
-The tutorials require a Linux or Unix environment with Docker installed.
+This first tutorial requires a Linux, Unix, or Mac environment with Docker installed.
 The examples run on a single host so that you can run them on your laptop or in a Virtual Machine.
 You are encouraged to try the examples using a `config.alloy` file and experiment with the examples yourself.
 
-To run the examples, you should have an {{< param "PRODUCT_NAME" >}} binary available. You can follow the instructions on how to [Install {{< param "PRODUCT_NAME" >}} as a Standalone Binary][install] to get a binary.
+## Install {{% param "PRODUCT_NAME" %}} and start the service
+
+### Linux
+
+Follow the instructions on the [Linux Install] page for the steps for several popular
+Linux distributions.  
+
+{{< admonition type="tip" >}}
+Make sure to follow the optional install step to enable the UI, we will be referring to it in this tutorial.
+{{< /admonition >}}
+
+Once you have completed this, follow the instructions to [Run on Linux][] using `systemctl`.
+
+### macOS
+
+Follow the instructions on the [macOS Install] page for Homebrew instructions. Once you have
+completed this, follow the instructions to [Run on macOS] which will start
+{{< param "PRODUCT_NAME" >}} as a Homebrew service.
 
 ## Set up a local Grafana instance
 
-You can use the following Docker Compose file to set up a local Grafana instance alongside Loki and Prometheus which are pre-configured as datasources. You can run and experiment with the examples on your local system.
+You can use the following Docker Compose file to set up a local Grafana instance alongside Loki and Prometheus which are pre-configured as datasources. You can run and experiment with the examples on your local system. In this tutorial,
+{{< param "PRODUCT_NAME" >}} will report logs to the loki running in this stack, and we can use Grafana to query and
+visualize them.
```

Comment:
I propose we replace this section with the following. The other info included in the original section has been relocated to other parts of the document for a better flow and ease of understanding. 

To enable Grafana Alloy to write data to Loki running in the local Grafana Stack, we will use the following Docker Compose file to set up a local Grafana instance alongside Loki and Prometheus which are pre-configured as data sources.